src/lib/stream_chooser.js

```diff
@@ -26,8 +26,12 @@ export default class StreamChooser {
     this.bandwidthInKbs = bandwidthInKbs;
   }
 
+  hlsVideos() {
+    return getHlsFormats(this.videos);
+  }
+
   bestVideo() {
```

Comment:
Just a random thought, but: I don't know this library well, so I don't know if this is best, but what about renaming this function to something like `bestNonHlsVideo`, or `bestStandardVideo`, or something like that, which makes the distinction with HLS more clear? (Assuming I haven't misunderstood the changes, ofc 😆 )

Reply:
Oh yes, that's way better! I went with `bestStandardVideo` and also updated `hlsVideos` to `hlsVideo`.